workflows/VGP-assembly-v2/kmer-profiling-hifi-VGP1/kmer-profiling-hifi-VGP1-tests.yml

```diff
@@ -12,22 +12,18 @@
     Ploidy: 1
   outputs:
     GenomeScope linear plot:
-      class: File
```

Comment:
@jmchilton do we like this ? It's a great union discriminator that would give us cleaner validation reports, I think ?

Reply:
The inputs you could argue are files. These are datasets - definitely not files. But the way CWL uses the word ``File`` is also just a real problem for my literal brain. We've already given into the syntax so why not just continue I guess. It is hard to appreciate consistency and technical quality of something that is wrong 😆. We can make this required in the best practice schema I think 😿. Will you give me the ``that`` syntax on the same argument about having a good clean discriminator 😆?

Reply:
Sure, that's fine. I guess we're still at the stage of defining what works, and then next step consolidating into something we like ?

Reply:
https://github.com/galaxyproject/galaxy/pull/18889